.github/workflows/macos.yml

```diff
@@ -0,0 +1,431 @@
+                 # otherwise subsequent commands will swallow bad exit status.
+          case "${RUNNER_OS:-Linux}" in
+          Linux)
+            if [ "$CHECK_REUSE" = "YES" -o "$CHECK_REUSE" = "ONLY" ]; then
+              echo "Calling reuse lint."
+              reuse lint
+            fi
+            if [ "$CHECK_REUSE" != "ONLY" -a "$CHECK_MKVK" != "ONLY" ]; then
+              if [ "$WASM_BUILD" = "YES" ]; then
+                ./scripts/build_wasm_docker.sh
+              else
+                ./scripts/build_linux.sh
+              fi
+            fi
+            ;;
+          macOS)
+            if [ "$CHECK_REUSE" != "ONLY" -a "$CHECK_MKVK" != "ONLY" ]; then
+              if [ "$PLATFORM" = "macOS" ]; then
+                ./scripts/build_macos.sh
+              else
+                ./scripts/build_ios.sh
+              fi
+            fi
+            ;;
+          esac
+          if [ "$CHECK_MKVK" = "ONLY" ]; then
+            scripts/check_mkvk.sh
+          fi
+          if [ "$DEPLOY_DOCS" = "YES" ]; then
+            # Some files in the pyktx docs have an _ prefix so Jekyll will
+            # not copy them from the gh-pages branch to the website. This
+            # file says no Jekyll files here. Treat all as ordinary files.
+            touch $BUILD_DIR/docs/html/.nojekyll
+          fi
+          set +e
+
+      - name: before_deploy
+        run: |
+          gem install octokit:'~> 4.15.0' mime-types
+          case "${RUNNER_OS:-Linux}" in
+          Linux)
+            ;;
+          macOS)
+            # Only notarize if we're going to deploy. I.e on a tag. Also check for
+            # access to signing certs to make sure we don't try this on a fork which
+            # has a tag.
+            #
+            # Although undocumented, except for a blog post, before_deploy is run for
+            # each deployment so we need to take steps to prevent notarization being
+            # run twice. https://github.com/travis-ci/travis-ci/issues/2570 about
+            # this has been open for years. What a crock!
+            if [ -n "$MACOS_CERTIFICATES_P12" -a -n "$TRAVIS_TAG" -a -z "$NOTARIZED" ]; then
+              ./scripts/notarize.sh $BUILD_DIR/KTX-Software-*.pkg $APPLE_ID $DEVELOPMENT_TEAM $ALTOOL_PASSWORD && export NOTARIZED="true"
+            fi
+            ;;
+          esac
+          if [ ! -f $REL_DESC_FILE -a -f RELEASE_NOTES.md ]; then
+            # Remove lines that are unnecessary in the context of a GitHub
+            # release description.
+            awk '/^Release Notes/,/^## Version/ { next }
+                 ! /<!-- Copyright/ && ! /<!-- SPDX/ { print }' RELEASE_NOTES.md \
+            > $REL_DESC_FILE
+            # dpl v2 alternative
+            #RELNOTES=$(awk '/^Release Notes/,/^## Version/ { next }
+            #     ! /<!-- Copyright/ && ! /<!-- SPDX/ { print }' RELEASE_NOTES.md)
+          fi
+
+      - name: Upload pages artifact
```

Comment:
Don't need docs deployment in this workflow. That is now done in docs.yml. Still need to build the docs in this though for inclusion in the built packages.